doc/engine/What's-new.md

```diff
@@ -1,190 +1,235 @@
+  They will be replaced with HDF5-based versions in future releases of the
+  OpenQuake Engine. For some calculator the HDF5-based implementation is
+  already available in the engine and can be accessed by using the ``--lite``
+  flag. For instance the classical hazard calculator is supported and can be
+  run with
+  
+  ``$ oq-engine --lite --run job.ini``
+  
+  However the calculators accessible with the ``--lite`` flag should be
+  considered experimental, previews of things to come, and they are still
+  subject to change.
+
+3. The new calculators can run with a single configuration file containing both
+the hazard and the risk parameters. This is actually the recommended way to
+run such calculators.
+
+6. The `scenario_risk` and `scenario_damage` calculators now support multiple
+GSIMs at the same time. This new functionality is documented in the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/).
+
+8. The `scenario_damage` calculator now supports multiple loss types at
+the same time, just as the `scenario_risk` calculator. Again, see the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/) for
+the details.
+
+7. The `event_based_risk` calculator has been optimized and enhanced. Now it
+is possible to generate the full event loss table for each
+asset and for each realization. Just set the configuration parameter
+`asset_loss_table=True` (the default is False).
+
+3. OpenQuake 1.6 supports officially the format NRML 0.5 for all risk
+models. In OpenQuake 1.5 it was supported in a limited and experimental way
+only for a subset of the vulnerability functions. Now all kind of risk
+models are supported: vulnerability models, fragility risk models and
+consequence models. Consequence models are brand new, introduced for
+the first time in this release. All of that is documented in the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/).
+The new format is simpler than before
+and more convenient to use, since the OpenQuake platform offers a web
+tool to prepare risk models in NRML 0.5. Beware that the web tool does
+not support validation of the risk models yet.
+
+4. The validation of the risk models in the engine has been
+improved. In particular now an user confusing a fragility model with a
+vulnerability model or a consequence model or any other combination
+will get a clear error message. Moreover, each risk model has a
+`lossCategory` attribute which must be set consistently with the name
+of the key in the job.ini file (see the [manual]
+(http://www.globalquakemodel.org/openquake/support/documentation/engine/)
+for the details).
+
+6. Some work has been going on hazardlib, as usual, and you can
+have a look at the [changelog](https://github.com/gem/oq-hazardlib/blob/engine-1.6/debian/changelog). The most prominent feature is the introduction of
+epistemic uncertainties on the fault geometry into the Source Model Logic Tree.
+
```

Comment:
I think we can add more detail here given the size of the change and the significant increase in flexibity to which the users are now exposed. I suggest something like:

6. Addition of support for new epistemic uncertaities in the source model logic tree. Users are now able to represent alternative geometries for the fault source typologies using the following uncertainty model definitions:

i. `simpleFaultGeometryAbsolute` - Replaces the simple fault geometry with the specified simple fault surface.
ii. `complexFaultGeometryAbsolute` - Replaces the complex fault geometry with the specified complex fault surface.
iii. `characteristicFaultGeometryAbsolute` - Replaces the characteristic fault geometry with the specified surface.
iv. `simpleFaultDipRelative` - Increases or decreases the fault dip with the specified value.
v. `simpleFaultDipAbsolute` - Replaces the simple fault dip with the specified value.

Also added is the ability to replace a given evenly discretised magnitude frequency distribution of a source in the logic tree using the `incrementalMFDAbsolute` option.

Documentation on the usage of these features will be added in the forthcoming version of the manual.